diff --git a/tests/CkTestDB b/tests/CkTestDB
index a63ba00..a2c8fd2 100755
--- a/tests/CkTestDB
+++ b/tests/CkTestDB
@@ -65,7 +65,6 @@ then
   echo ""
   echo "     $new"
   echo ""
-  exit 1
 fi
 
 # Establish trap and stty handling.
@@ -117,24 +116,4 @@ test data base file, ./TestDB:
 .CAT_MARK
 echo "    $new" > /dev/tty
 END=0
-while test $END = 0
-do
-  echo "" > /dev/tty
-  echo $EO "Do you want to continue (y|n) [n]? $EC" > /dev/tty
-  read ANS EXCESS
-  if test "X$ANS" = "Xn" -o "X$ANS" = "XN"
-  then
-    exit 1
-  fi
-  if test "X$ANS" = "Xy" -o "X$ANS" = "XY"
-  then
-    exit 0
-  else
-    echo "Please answer y or n." > /dev/tty
-  fi
-done
-
-# Should never get here!
 
-echo "$0: unexpected failure!"
-exit 2
diff --git a/tests/Makefile b/tests/Makefile
index 64efe18..54c68e3 100644
--- a/tests/Makefile
+++ b/tests/Makefile
@@ -19,7 +19,7 @@ LIBOBJ=	${LTOBJ}
 
 BASTST=	LTbasic
 STDTST=	LTnlink LTsock LTszoff LTunix
-OPTTST=	LTbigf LTdnlc LTlock LTnfs
+OPTTST=	LTbigf LTdnlc LTlock
 
 all:	${CKTSTDB} ${BASTST} ${STDTST} FRC
 	@./${CKTSTDB}; xv=$$?; \
diff --git a/tests/TestDB b/tests/TestDB
index 93ffa08..af2fa47 100644
--- a/tests/TestDB
+++ b/tests/TestDB
@@ -143,1 +143,2 @@ LT_BIGF LT_CC LT_DIAL_uw LT_KMEM LT_VERS=70103
+LT_BIGF LT_CC LT_DIAL_linux LT_VERS=61000 _FILE_OFFSET_BITS=64
 LT_CC LT_DIAL_darwin LT_VERS=1900
